resources/public/api/conf/1.0/application.raml

```diff
@@ -294,15 +294,261 @@ uses:
+      is: [headers.acceptHeader]
+      securedBy: [ sec.x-application ]
+      headers:
+        Authorization:
+          description: "OAuth 2.0 Bearer Token or server token."
+          type: string
+          required: true
+          example: Bearer <BEARER_TOKEN>
+        Content-Type:
+          description: "Specifies the content-type of the body."
+          type: string
+          required: true
+          example: application/json
+      queryParameters:
+        startYear:
+          description: "Starting Year of response"
+          type: integer
+          required: true
+          example: 2016
+        endYear:
+          description: "Ending year of response"
+          type: integer
+          required: true
+          example: 2019
+      body:
+        application/json:
+          schema: !include schemas/organisations/create-organisations-taxpayer-detail.json
+          example: !include examples/organisations/create-organisations-taxpayer-detail.json
+      responses:
+        201:
+          description: The created taxpayer details
+          body:
+            application/json:
+              type: !include schemas/organisations/create-organisations-taxpayer-detail.json
+              example: !include examples/organisations/create-organisations-taxpayer-detail.json
+        400:
+          body:
+            application/json:
+              type: types.errorResponse
+              examples:
+                BadRequest:
+                  description: Invalid request
+                  value:
+                    code: INVALID_REQUEST
+
+  /organisations/corporation-tax/{utr}/return/details:
+    uriParameters:
+      utr:
+        description: "A Unique Tax Reference"
+        type: string
+        required: true
+        example: "1234567890"
+    post:
+      displayName: "Create a corporation-tax payload"
+      is: [headers.acceptHeader]
+      securedBy: [ sec.x-application ]
+      headers:
+        Authorization:
+          description: "OAuth 2.0 Bearer Token or server token."
+          type: string
+          required: true
+          example: Bearer <BEARER_TOKEN>
+        Content-Type:
+          description: "Specifies the content-type of the body."
+          type: string
+          required: true
+          example: application/json
+      queryParameters:
+        startYear:
+          description: "Starting Year of response"
+          type: integer
+          required: true
+          example: 2016
+        endYear:
+          description: "Ending year of response"
+          type: integer
+          required: true
+          example: 2019
+      body:
+        application/json:
+          schema: !include schemas/organisations/create-corporation-tax-detail.json
+          example: !include examples/organisations/create-corporation-tax-detail.json
+      responses:
+        201:
+          description: The created corporation-tex details
```

Comment:
tex? 

Perhaps rename to `corporation-tax  return details`

Reply:
Good typo spot xD